doc/themes/the-things-stack/assets/css/theme.scss

```diff
@@ -1,13 +1,15 @@
 @charset "utf-8";
 
 @import url('https://fonts.googleapis.com/css?family=Lato:400,700&display=swap');
+@import url('https://rsms.me/inter/inter.css');
```

Comment:
If we're _switching_ to Inter, we can just remove Lato.

Reply:
Why are the fonts sourced from remote anyway? Shouldn't this be hosted locally rather? Why now sort of do both (with Lato) which I don't understand.

Reply:
are we switching to Inter? @pierrephz @kschiffer ? i think Inter looks better but you guys are the designers

Reply:
We are eventually, but this needs to be done in a more organized way. Please see: https://discuss.thethingsindustries.com/t/the-things-logos-redesign/331/8

So that actually means that we need to stick to the old fonts for the time being.

cc @pierrephz